app/pages/app/app.js

```diff
@@ -343,6 +344,21 @@ export class AppComponent extends React.Component {
     return null;
   }
 
+  renderNavHeader() {
+    if (!this.isPatientVisibleInNavbar()) return null;
+
+    const { patient, userIsCurrentPatient } = this.props;
+    const trackMetric = this.props.context.trackMetric;
```

Comment:
I'm wondering if we have explored ways to make trackMetric accessible globally to reduce prop drilling, but i'm not familiar with the nuances of the function. I feel like it could be achievable with React's Context API

Reply:
We haven't explored much, and it certainly is a relic of the past that could be improved through use of Context and/or a custom hook. We do a _lot_ of prop drilling that's fairly out of modern best practices at this point. Might need to consider someplace to put these sorts of code quality improvement suggestions so they don't get lost.